### PR TITLE
Fixes #27991 - Decreasing precision in Task Progress

### PR DESCRIPTION
**Pull request by sseelam2**
This PR changes the precision of the individual task progress. The progress bar is accompanied by a numeric value as "n% Complete". (i.e.: 10.06661732050331% Complete). Reduced the precision from 14 to 2.

### CODE REVIEW

**Comment by chris1984** [comment]
[test redmine]

**Comment by chris1984** [comment]
[test]

File: webpack/ForemanTasks/Components/TaskDetails/TaskDetailsSelectors.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const selectErrors = state => {
 };
 
 export const selectProgress = state =>
-  selectTaskDetails(state).progress * 100 || 0;
+    selectTaskDetails(state).progress ? selectTaskDetails(state).progress.toFixed(4) * 100 : selectTaskDetails(state).progress * 100 || 0;
```
Comment (adamruzicka) [inline webpack/ForemanTasks/Components/TaskDetails/TaskDetailsSelectors.js]: Could you keep the indentation consistent?

File: webpack/ForemanTasks/Components/TaskDetails/TaskDetailsSelectors.js [hunk 2]
```diff
@@ -29,7 +29,9 @@ export const selectErrors = state => {
 };
 
 export const selectProgress = state =>
-  selectTaskDetails(state).progress * 100 || 0;
+  selectTaskDetails(state).progress
+    ? selectTaskDetails(state).progress.toFixed(4) * 100
+    : selectTaskDetails(state).progress * 100 || 0;
```
Comment (tbrisker) [inline webpack/ForemanTasks/Components/TaskDetails/TaskDetailsSelectors.js]: This trenary branch will only be reached if    `selectTaskDetails(state).progress` is undefined or 0 - so we can fall back to 0 directly
```suggestion
    : 0;
```

**Comment by jturel** [comment]
This looks good to me. Anything preventing merge?

**Comment by adamruzicka** [comment]
Thank you @sseelam2 and @jturel !

**Pull request merged**